app/models/orders_package.js

```diff
@@ -12,7 +12,7 @@ export default Model.extend({
   sentOn: attr('date'),
   state: attr('string'),
 
-  package:    belongsTo('package', { async: false }),
+  package:    belongsTo('package', { async: true }),
```

Comment:
@steveyken , when async is set to false and if you are loading data of that object then it doesn't load child/parent records, if on detail page you're only showing records for that Object then you're fine but if you are also showing details like **Item.packages** then in this case it'll break because associated packages record with Item are not loaded.
- If you set async to true, then whenever you're firing request for any model Object it will fire addition request to load related child/parent models.
- So in a page where you're showing detail of item's with child packages it async: true will work because packages are already loaded otherwise it'll fail saying that **Associated records are not loaded**